dashboard/app/views/levels/_admin.html.haml

```diff
@@ -36,7 +36,7 @@
         %li (Cannot edit)
 
     %br/
-    This level is in #{@level.script_levels.count} scripts:
+    This level is in #{@level.script_levels.to_a.count} scripts:
```

Comment:
Again, I'll note that this is unrelated (for the benefit of future readers of this PR).